src/containers/my-resources/categories-container/CategoriesContainer.tsx

```diff
@@ -53,6 +59,20 @@ const CategoriesContainer = () => {
     [setAlert]
   )
 
+  const onResponse = useCallback(
+    (response: Categories | null) => {
+      const categoryName = response ? response.name : ''
+
+      setAlert({
+        severity: snackbarVariants.success,
+        message: t('myResourcesPage.categories.successCreation', {
```

Comment:
inside snackbar-context it passes this message param to translation function: t(message), here you pass this message already with translation, so will it work?

Reply:
> inside snackbar-context it passes this message param to translation function: t(message), here you pass this message already with translation, so will it work?

yeah, it's already working, you can see it in preview